.github/workflows/update-color-primitives.yml

```diff
@@ -48,7 +48,11 @@ jobs:
           EOF
           done
 
-      - run: make fmt
+      - uses: JohnnyMorganz/stylua-action@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          version: latest
+          args: -f stylua.toml --verify -- '${{ _DEST_DIR }}'
```

Comment:
I decided to use this action instead of the nix actions. Let me know if you'd like me to change it to use the nix actions instead.

This action is authored/maintained by the author/maintainer of stylua; it installs and then caches the stylua binary directly from GitHub releases. The stylua binary is only around 3MB. 

> **Note** It may be worth changing the nix actions in the other workflows to use this action as well (at least in the cases where only stylua is being downloaded and ran). I'm not totally sure, but the nix actions appear to be [downloading alot more than just stylua and seem to take ~20 seconds to run](https://github.com/projekt0n/github-nvim-theme/actions/runs/5094878837/jobs/9159217525) instead of just 2 or 3.